.github/workflows/ci.yaml

```diff
@@ -205,6 +202,10 @@ jobs:
         env:
           REPO_NAME: ${{ steps.ref.outputs.repo-name }}
 
+      - id: setup-buildx
+        name: Set up Docker Buildx
```

Comment:
Why do we need this?

Reply:
I saw from the error in this run: https://github.com/hms-dbmi-cellenics/ui/runs/7291171952?check_suite_focus=true and read about it from the `docker/build-push-action` section about driver: https://github.com/docker/build-push-action#usage